bundle/tests/scorecard/config.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
+kind: Configuration
+metadata:
+  name: config
+stages:
+- parallel: true
+  tests:
+  - entrypoint:
+    - scorecard-test
+    - basic-check-spec
+    image: quay.io/operator-framework/scorecard-test:v1.11.0
```

Comment:
This is 3 revs back. Can we change these to v1.14.0?

Reply:
replaced across multiple yamls